development/generate-icon-names.js

```diff
@@ -39,7 +39,9 @@ const generateIconNames = async () => {
         getIconNameKebabCase(fileName)),
   );
 
-  return iconNames;
+  const iconNamesStringified = JSON.stringify(iconNames);
+
+  return iconNamesStringified;
 };
 
 module.exports = { generateIconNames };
```

Comment:
Updating `generateIconNames` to be synchronous because @Gudahtt somehow figured out that to get it to work with jest 

> I was able to get it to work by making generateIconNames synchronous (swapping out fs.promises.readdir with fs.readdirSync), and calling it in test.env.js. Must be some Jest bug/quirk about async setup, I don't understand what's going on there,